DataConnectors/ESETPROTECT/Connector_Syslog_ESETPROTECT.json

```diff
@@ -0,0 +1,154 @@
+                                "description": "Select the machine to install the agent on and then click **Connect**.",
+                                "instructions": [
+                                    {
+                                        "parameters": {
+                                            "linkType": "InstallAgentOnLinuxVirtualMachine"
+                                        },
+                                        "type": "InstallAgent"
+                                    }
+                                ]
+                            },
+                            {
+                                "title": "Install agent on a non-Azure Linux Machine",
+                                "description": "Download the agent on the relevant machine and follow the instructions.",
+                                "instructions": [
+                                    {
+                                        "parameters": {
+                                            "linkType": "InstallAgentOnLinuxNonAzure"
+                                        },
+                                        "type": "InstallAgent"
+                                    }
+                                ]
+                            }
+                        ]
+                    },
+                    "type": "InstructionStepsGroup"
+                }
+            ]
+        },
+        {
+            "title": "2. Configure the logs to be collected",
+            "description": "Configure the facilities you want to collect and their severities.\n\n1.  Under workspace advanced settings **Configuration**, select **Data** and then **Syslog**.\n2.  Select **Apply below configuration to my machines** and select the facilities and severities. The default ESET PROTECT facility is **user**.\n3.  Click **Save**.",
+            "instructions": [
+                {
+                    "parameters": {
+                        "linkType": "OpenAdvancedWorkspaceSettings"
+                    },
+                    "type": "InstallAgent"
+                }
+            ]
+        },
+        {
+            "title": "3. Configure ESET PROTECT",
+            "description": "Configure ESET PROTECT to send all events thorugh Syslog.\n\n1.  Follow [these instructions](https://help.eset.com/protect_admin/latest/en-US/admin_server_settings_syslog.html) to configure syslog output. Make sure to select **BSD** as the format and **TCP** as the transport.\n\n2.  Follow [these instructions](https://help.eset.com/protect_admin/latest/en-US/admin_server_settings_export_to_syslog.html) to export all logs to syslog. Select **JSON** as the output format.",
+            "instructions": [
+                {
+                    "parameters": {
+                        "linkType": "OpenAdvancedWorkspaceSettings"
+                    },
+                    "type": "InstallAgent"
+                }
+            ]
+        }
+    ]
+}
```

Comment:
Please add a Metadata object within this file. See https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_REST_API_template.json for further detail.